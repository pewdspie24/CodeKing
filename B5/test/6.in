74
3 18
2 13
9 11
8 15
7 17
10 12
8 14
6 12
6 16
6 10
5 12
3 12
3 10
6 17
5 11
10 13
4 14
1 12
9 13
3 17
9 11
3 18
4 12
8 12
7 15
7 10
4 13
8 16
9 16
10 13
10 11
1 17
2 11
5 17
5 11
3 13
2 14
2 13
1 15
3 14
9 13
8 15
10 14
6 14
1 10
2 12
1 13
7 16
6 14
2 10
2 16
4 17
8 13
8 12
5 15
2 14
6 18
3 15
2 18
2 16
7 15
9 11
3 15
1 17
2 14
6 11
1 18
7 13
2 13
1 15
8 14
7 12
2 16
2 17
