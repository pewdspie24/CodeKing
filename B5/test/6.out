362797056
470184984576
379749833583241
13841287201
531441
4398046511104
470184984576
470184984576
35184372088832
4096
48828125
1073741824
131072
244140625
282475249
1000000000000000
100000000000000000
65536
34522712143931
1977326743
362797056
2357947691
505447028499293771
4747561509943
1953125
268435456
152587890625
4747561509943
100000000000000
1024
1000000000000
177147
34522712143931
1000000000000000
2251799813685248
512
362797056
60466176
16677181699666569
40353607
2048
1977326743
19683
678223072849
22876792454961
4096
129140163
100000000000
40353607
35184372088832
60466176
1073741824
762939453125
25937424601
505447028499293771
177147
